### PR TITLE
FC-0001: Remove enterprise_catalogs API endpoint

### PR DESCRIPTION
**Pull request by ruzniaievdm**
(5 of 5) for closes https://github.com/openedx/public-engineering/issues/61

## Description:
Removed the [enterprise_catalogs](https://github.com/openedx/edx-enterprise/blob/decc608453f733ba53203345de9b0af842d60f3b/enterprise/api/v1/views.py#L1003) endpoint

## Linked PRs which need merge before merging this:
- [ ] https://github.com/openedx/edx-enterprise-data/pull/313
- [x] https://github.com/openedx/enterprise-catalog/pull/468
- [x] https://github.com/openedx/frontend-app-admin-portal/pull/801
- [ ] https://github.com/openedx/ecommerce/pull/3742

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ruzniaievdm! I've created [OSPR-6792](https://openedx.atlassian.net/browse/OSPR-6792) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by johnnagro** [comment]
I am under the impression that these v1 endpoints are still in use by customers and 3rd party integrations. @mattdrayer may have some more information.

**Comment by mattdrayer** [comment]
Has a usage analysis been conducted by edX/2U to see which endpoints are actively being utilized, and the customers/integrators notified of the planned deprecation/decommission of the V1 endpoints?

**Comment by mattdrayer** [comment]
The good news is that the external consumers SHOULD be hitting the V1 endpoints via the AWS API Gateway, and so as long as the mapping of those V1 endpoints still returns a valid response then it does not matter which Open edX component(s) service those requests.

File: api-compact.yaml [hunk 1]
```diff
@@ -300,83 +292,6 @@ x-amazon-apigateway-integration-with-enterprise-customer-catalog-uuid-and-course
 endpoints:
   v1:
 
-    # /v1/enterprise_catalogs/
```
Comment (mattdrayer) [inline api-compact.yaml:303]: I don't think we should be stripping out the API gateway V1 endpoints -- instead they should be mapped to new backend endpoints and continue to provide responses according to their current formats/schemas.

**Comment by mphilbrick211** [comment]
@feanil - just checking in on this - do you know if this is still in progress? Does it just need to be reviewed?

**Comment by feanil** [comment]
@mphilbrick211 no, this is not still in progress, unfortunately we were not able to get this work landed and I don't think there is enough appetite for this at the moment so I'm gonna close this for now and it can be revisited in the future.

**Comment by openedx-webhooks** [comment]
@ruzniaievdm Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-enterprise&entry.867055334=https://github.com/openedx/edx-enterprise/pull/1582&entry.1484655318=https://github.com/ruzniaievdm&entry.752974735=2022-06-13+15:11&entry.1917517419=2023-01-27+18:30&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**